app/src/main/AndroidManifest.xml

```diff
@@ -30,8 +30,8 @@
 
                 <!--Add in your scheme/host from registered redirect URI-->
                 <data android:scheme="msauth"
-                    android:host= "YOUR_PACKAGE_NAME - must be registered at https://aka.ms/MobileAppReg"
-                    android:path= "/YOUR_DECODED_SIGNATURE_HASH - must be registered at https://aka.ms/MobileAppReg>" />
+                    android:host= "com.azuresamples.msalandroidapp"
```

Comment:
I think it was on purpose to keep it in this way. 
```
android:host= "YOUR_PACKAGE_NAME - must be registered at https://aka.ms/MobileAppReg"
android:path= "/YOUR_DECODED_SIGNATURE_HASH - must be registered at https://aka.ms/MobileAppReg>" />
```

Reply:
Yes.  I'll put back the original values... prior to actually merging.  Leaving it this way for testing.